### ISSUE TITLE
Add support for editing AudioClip and VideoClip

### ISSUE THREAD

**Issue by timius100** [nesrak1/UABEA#136]
Hi, can you add the plugin for editing AudioClip or VideoClip?

**Comment by karohan** [comment]
I wish something like this was possible :(

**Comment by dwstkar490-afk** [comment]
Hello,
Is it possible to import sound or not?
And I have a question, maybe you can help me.
It’s about the .bundle file, whose full name is audioassets_assets_all_a2142e0d53745603f2564d67d96220e7.bundle, which is specifically for sound.
Its folder path is:
Commandos 2 - HD Remaster\Commandos 2 HD Remaster_Data\StreamingAssets\aa\Windows\StandaloneWindows64
Engine Version: 2018.4.22f1

I imported the sound into it using UnityEX. I opened the file with various tools and confirmed the sound was inside. But when the game goes to the main menu and I click 'Start Game', the screen turns black and it doesn't progress further.
Do you know the reason for this?

**Comment by nesrak1** [comment]
Have you read this? https://github.com/nesrak1/UABEA?tab=readme-ov-file#addressables

> Is it possible to import sound or not?

It's been explained many times in linked issues, but the TLDR again is that sound importing isn't supported because I don't want to use the official fmod API (licensing, native library, etc.) and alternative options are not great. Additionally, there needs to be infrastructure for modifying resS/resource files which does not yet exist. I don't have much motivation to do so at the moment.

**Comment by dwstkar490-afk** [comment]
I hadn't read it.
But my question was general, not just about your program, but whether this is possible in any program that can be found.
I mean, is it possible for me to import a sound file into the game under the conditions mentioned and have it work?
I am looking for a program or a method to do this.

Is there a way to add the ability to import sound files to your program without encountering the problems you are facing?

**Comment by nesrak1** [comment]
I don't know what you mean "under the conditions" because it doesn't look like you listed any conditions. You said you imported the audioclip with unityex. This tool should probably be replacing unityex entirely, but if it works for you right now, that's fine for now. If you do the catalog patching (which btw, I have been told unityex has their own catalog patcher too but I'm not going to pay for unityex to find out), it most likely solves your problem.

TLDR: importing audio was probably successful, CRC checking is now your next issue

**Comment by dwstkar490-afk** [comment]
Maybe it's unrelated, but can you help me create a .fsb file exactly like TUT04_B_AR_1_00007.fsb that I'm sending?
https://www.mediafire.com/file/99iqgel69k84v22/TUT04_B_AR_1_00007.fsb/file

**Comment by SpiderTranslate** [comment]
> ฉันไม่เข้าใจว่าคุณหมายถึง "ภายใต้เงื่อนไข" อะไร เพราะดูเหมือนคุณจะไม่ได้ระบุเงื่อนไขใดๆ คุณบอกว่าคุณนำเข้าคลิปเสียงด้วย UnityEx เครื่องมือนี้ควรจะเข้ามาแทนที่ UnityEx ทั้งหมด แต่ถ้ามันใช้งานได้สำหรับคุณในตอนนี้ ก็ถือว่าโอเคแล้ว ถ้าคุณทำการแก้ไขแคตตาล็อก (ซึ่งจริงๆ แล้ว ฉันได้ยินมาว่า UnityEx ก็มีเครื่องมือแก้ไขแคตตาล็อกของตัวเองเช่นกัน แต่ฉันจะไม่จ่ายเงินให้ UnityEx เพื่อลองใช้ดู) มันน่าจะแก้ปัญหาของคุณได้
> 
> สรุปสั้นๆ: การนำเข้าไฟล์เสียงน่าจะสำเร็จแล้ว ปัญหาต่อไปคือการตรวจสอบ CRC

Could you possibly update the tools to support video clip import and export? Or, do you happen to remember which version included this feature? I recall seeing it in a previous build, but I've been trying to find it and can't seem to locate it anywhere.

This is quite important because, in many FMV games, the dialogue audio isn't stored as separate sound files—it's embedded directly within the video clips

**Comment by dwstkar490-afk** [comment]
@nesrak1 
Sorry, I think I started off on the wrong foot. Let's start over from the beginning.
What problem are you having, and how can I help resolve it?
